### DISCUSSION TITLE
rectangle设置不生效

### DISCUSSION THREAD

**Discussion by yadanchina** [zouyaoji/vue-cesium#511] [Q&A]
图层设置rectangle后，会出现拉伸效果，或者不生效的问题，例如我设置一个矩形位置之后，页面还是会请求矩形之外的瓦片数据

**Comment by zouyaoji** [comment]
重现不了哈，请提供能复现的例子。
![image](https://github.com/zouyaoji/vue-cesium/assets/24430116/903c1dcb-415a-428e-9da8-f8b48b2420bd)

**Reply by yadanchina** [reply]
网上搜索到的是，如果给基础图层添加rectangle就是会拉伸，只有一个图层设置矩形范围就会拉伸占据非矩形范围的空间

例如我设置一个矩形位置之后，页面还是会请求矩形之外的瓦片数据，
wms图层服务加载，设置矩形范围后，不生效还是控制台内会加载到

背景：本地的地图服务有几个 地方没有数据，然后发送这几个地方的数据会导致瓦片数据找不到，然后地图服务会崩掉，所以需要避免加载这几个地方的数据，设置矩形区域之后，还是可以在控制台内看到对应的请求。会导致地图服务崩掉。

是内网应用，外网暂时无法浮现
